README.md

```diff
@@ -175,6 +175,24 @@ Volume snapshot APIs and client library are now in a separate sub-module: `githu
 
 Use the command `go get -u github.com/kubernetes-csi/external-snapshotter/client/v3@v3.0.0` to get the client library.
 
+### Setting Quota limits with Snapshot custom resources
+[`ResourceQuotas`](https://kubernetes.io/docs/concepts/policy/resource-quotas/) are namespaced objects that can be used to set limits on objects of a particular [`Group.Version.Kind`](https://book.kubebuilder.io/cronjob-tutorial/gvks.html). Before we set resource quota, make sure that snapshot CRDs are installed in the cluster. If not please follow [this guide](https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/docs/deploy-1.17-and-later.md)
+```
+kubectl get crds | grep snapshot
+```
+
+Now create a `ResourceQuota` object which sets the limits on number of volumesnapshots that can be created:
+```yaml
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: snapshot-quotas
+spec:
+  hard:
+    count/volumesnapshots.snapshot.storage.k8s.io: "10"
+    count/volumesnapshotclasses.snapshot.storage.k8s.io: "10"
```

Comment:
VolumeSnapshotClass is non-namespaced objects.  Isn't resource quota only for namespaced objects?

Reply:
updated.